server/pulp/server/async/tasks.py

```diff
@@ -136,7 +136,7 @@ def _get_unreserved_worker():
     # Build a mapping of queue names to Worker objects
     workers_dict = dict((worker['name'], worker) for worker in Worker.objects())
     worker_names = workers_dict.keys()
-    reserved_names = [r['worker_name'] for r in ReservedResource.get_collection().find()]
+    reserved_names = [r['worker_name'] for r in ReservedResource.objects()]
```

Comment:
I think `ReservedResource.objects()` this would read more clearly as `ReservedResource.objects.all()`